nodes/exchange/FCStd_read.py

```diff
@@ -0,0 +1,102 @@
+from sverchok.dependencies import FreeCAD
+from sverchok.utils.dummy_nodes import add_dummy
+
+if FreeCAD is None:
+    add_dummy('SvReadFCStdNode', 'SvReadFCStdNode', 'FreeCAD')
+
+else:
+    F = FreeCAD
+    import bpy,sys
+    from bpy.props import IntProperty, FloatProperty, StringProperty, BoolProperty
+    from sverchok.node_tree import SverchCustomTreeNode
+    from sverchok.utils.logging import info, exception
+    from numpy import ndarray
+
+    class SvReadFCStdNode(bpy.types.Node, SverchCustomTreeNode):
+        ''' SvReadFCStdNode '''
+        bl_idname = 'SvReadFCStdNode'
+        bl_label = 'Read FCStd'
+        bl_icon = 'IMPORT'
+        solid_catergory = "Inputs"
+        
+        read_update : BoolProperty(name="read_update", default=True)
+        part_filter : StringProperty(name="part_filter", default="", description="use ',' to separate name with no space: part1,part2,... ")
+        inv_filter : BoolProperty(name="inv_filter", default=False)

```

Comment:
If you want the node to update on property change you would need to get the `updateNode `function from data_structure as here:
https://github.com/nortikin/sverchok/blob/a95871e3a5a77b13e63fda4654a07a638f6ab19d/nodes/curve/apply_field_to_curve.py#L8
and add a update keyword to the property calling to that function
https://github.com/nortikin/sverchok/blob/a95871e3a5a77b13e63fda4654a07a638f6ab19d/nodes/curve/apply_field_to_curve.py#L30-L33